contrib/ros/envs/sawyer/example_launchers/trpo_sim_sawyer_pnp.py

```diff
@@ -0,0 +1,81 @@
+import os.path as osp
+
+from geometry_msgs.msg import Point
+import numpy as np
+import rospy
+
+from rllab.baselines.linear_feature_baseline import LinearFeatureBaseline
+from rllab.envs.normalized_env import normalize
+from rllab.misc.instrument import run_experiment_lite
+
+from sandbox.rocky.tf.algos.trpo import TRPO
+from sandbox.rocky.tf.policies.gaussian_mlp_policy import GaussianMLPPolicy
+from sandbox.rocky.tf.envs.base import TfEnv
+
+import contrib.ros.envs.sawyer
+from contrib.ros.envs.sawyer.pick_and_place_env import PickAndPlaceEnv
+from contrib.ros.envs.task_object_manager import TaskObject, TaskObjectManager
+
+
+def run_task(*_):
+    model_dir = osp.join(
+        osp.dirname(contrib.ros.envs.sawyer.__file__), 'models')
```

Comment:
Maybe you can make this a constant in `contrib.ros.envs.sawyer` in `__init__.py`